### DISCUSSION TITLE
Error when checking for updates

### DISCUSSION THREAD

**Discussion by beho** [MiKTeX/miktex#1268] [Q&A]
Hi,

I've installed latest Miktex on Windows machine in system-wide mode and I'm now trying to check for updates from Miktex console. But I'm getting follwing error:

```
GENERAL MIKTEX INFORMATION
ReportDate: 2023-01-17 17:45:49
CurrentVersion: 22.10
SetupDate: 2023-01-17 17:43:00
SetupVersion: 22.10
Configuration: Regular
GitInfo: f9aad01 / 2022-10-15 09:37:00
OS: Windows 6.3.9600
SharedSetup: yes
LinkTargetDirectory: C:\Program Files\MiKTeX\miktex\bin\x64
PathOkay: yes
LastUpdateCheckAdmin: not yet
LastUpdateAdmin: not yet
LastUpdateDbAdmin: not yet
SystemAdmin: yes
RootPrivileges: yes
AdminMode: yes
Root0: C:\ProgramData\MiKTeX
Root1: C:\Program Files\MiKTeX
CommonInstall: C:\Program Files\MiKTeX
CommonConfig: C:\ProgramData\MiKTeX
CommonData: C:\ProgramData\MiKTeX
Invokers: 

The following issues were detected:
  1: minor issue: You are running MiKTeX on an unsupported version of Windows.
  2: major issue: So far, no MiKTeX administrator has checked for updates.

ERROR DETAILS
Program: miktex-console_admin
Source: Libraries\MiKTeX\PackageManager\CurlWebSession.cpp:466
Error: Error response from server: 444
Details: 
  responseCode: 444
```

I have no idea what I'm doing wrong. I need to install some packages but I'm unable to do so.

I'd be most grateful for any help.

Thanks!

**Comment by beho** [comment]
It is not happening today so it seems that was a temporary glitch.

**Comment by alexandresendai** [comment]
The same issue is still on going here. Do you have any suggestion, please, to solve the issue ?

**Comment by mcbaneg** [comment]
I have a similar problem.  Trying to install a package, or just to update, I'm getting "MiKTeX encountered an internal error."  The error-info clipboard gives

GENERAL MIKTEX INFORMATION
Date: 2023-02-11 17:09:14
MiKTeX: 2.9.7440
GitInfo: af89c1b / 2020-05-20 09:20:28
OS: Windows 10 Enterprise, 64-bit, build 19044
SharedSetup: yes
LinkTargetDirectory: C:\Program Files\MiKTeX 2.9\miktex\bin\x64
PathOkay: yes
LastUpdateCheckAdmin: 2020-06-27 11:01:34
LastUpdateAdmin: 2020-06-27 10:57:12
LastUpdateDbAdmin: 2020-06-27 11:01:34
SystemAdmin: yes
RootPrivileges: yes
AdminMode: yes
Root0: C:\ProgramData\MiKTeX\2.9
Root1: C:\localtexmf
Root2: C:\Program Files\MiKTeX 2.9
CommonInstall: C:\Program Files\MiKTeX 2.9
CommonConfig: C:\ProgramData\MiKTeX\2.9
CommonData: C:\ProgramData\MiKTeX\2.9
Invokers: 

The following issues were detected:
  1: minor issue: It has been a long time since a MiKTeX administrator has checked for updates.

ERROR DETAILS
Program: C:\Program Files\MiKTeX 2.9\miktex\bin\x64\miktex-console_admin.exe
Source: Libraries\MiKTeX\PackageManager\RestRemoteService.cpp:245
Error: MiKTeX encountered an internal error.

------------------
Also: When I try to change my package repository, the list of options is empty.  I presume this is related.  One possibility I considered was that the security setup wasn't letting the MikTeX manager access the network to look stuff up.  But I don't see any recent blocking activity in the Windows Security log.  Up until recently it was able to successfully do auto-install of packages in the normal way; I first encountered this problem a couple of days ago when I tried to use elsarticle.cls, which I don't have installed locally.

Suggestions appreciated.
Thank you!
-George.

**Comment by alexandresendai** [comment]
Hello George,
I recently succeeded to install the package by changing the time i was
doing it. Now, I try to load it in the evening and it works.. I don't know
why.
Thank you very much fr reaching me, it's very nice from you.
Best regards,
Alexandre

**Comment by cmtappu96** [comment]
I have a similar problem. Trying to check for package updates and I am getting 'MikTeX encoutered an internal error.'

GENERAL MIKTEX INFORMATION
Date: 2024-07-01 17:00:18
MiKTeX: 2.9.7250
OS: Windows 10 Home, 64-bit, build 22631
SharedSetup: yes
LinkTargetDirectory: C:\Program Files\MiKTeX 2.9\miktex\bin\x64
PathOkay: yes
LastUpdateCheckAdmin: 2019-12-03 22:35:00
LastUpdateAdmin: 2019-12-03 22:33:33
SystemAdmin: yes
RootPrivileges: no
AdminMode: no
Root0: C:\Users\cmtap\AppData\Roaming\MiKTeX\2.9
Root1: C:\Users\cmtap\AppData\Local\MiKTeX\2.9
Root2: C:\ProgramData\MiKTeX\2.9
Root3: C:\Program Files\MiKTeX 2.9
UserInstall: C:\Users\cmtap\AppData\Roaming\MiKTeX\2.9
UserConfig: C:\Users\cmtap\AppData\Roaming\MiKTeX\2.9
UserData: C:\Users\cmtap\AppData\Local\MiKTeX\2.9
CommonInstall: C:\Program Files\MiKTeX 2.9
CommonConfig: C:\ProgramData\MiKTeX\2.9
CommonData: C:\ProgramData\MiKTeX\2.9
Invokers: non-existing/explorer

Warning: the following problems were detected:
  1: It has been a long time since system-wide updates were checked.

ERROR DETAILS
Program: C:\Program Files\MiKTeX 2.9\miktex\bin\x64\miktex-console.exe
Source: Libraries\MiKTeX\PackageManager\RestRemoteService.cpp:245
Error: MiKTeX encountered an internal error.

MikTeX logs
2019-12-03 21:27:51,920-0500 INFO  main - starting: MiKTeX Console 2.9.7206 (MiKTeX 2.9.7250 64-bit)
2019-12-03 21:28:02,402-0500 INFO  main - finishing with exit code 0
2019-12-03 22:34:43,453-0500 INFO  main - starting: MiKTeX Console 2.9.7206 (MiKTeX 2.9.7250 64-bit)
2019-12-03 22:34:49,187-0500 INFO  main - finishing with exit code 0
2019-12-12 06:34:56,289-0500 INFO  main - starting: MiKTeX Console 2.9.7206 (MiKTeX 2.9.7250 64-bit)
2019-12-12 06:35:06,304-0500 INFO  main - finishing with exit code 0
2020-01-12 11:13:44,563+0530 INFO  main - starting: MiKTeX Console 2.9.7206 (MiKTeX 2.9.7250 64-bit)
2020-01-12 11:14:03,060+0530 INFO  main - finishing with exit code 0
2024-06-30 11:41:10,996-0400 INFO  main - starting: MiKTeX Console 2.9.7206 (MiKTeX 2.9.7250 64-bit)
2024-06-30 11:41:27,320-0400 INFO  main - finishing with exit code 0
2024-07-01 16:46:55,694-0400 INFO  main - starting: MiKTeX Console 2.9.7206 (MiKTeX 2.9.7250 64-bit)
2024-07-01 16:47:09,282-0400 INFO  main - finishing with exit code 0
2024-07-01 16:52:27,786-0400 INFO  main - starting: MiKTeX Console 2.9.7206 (MiKTeX 2.9.7250 64-bit)
2024-07-01 16:52:50,436-0400 INFO  mainwindow - checking for updates...
2024-07-01 16:52:51,133-0400 ERROR mainwindow - Libraries\MiKTeX\PackageManager\RestRemoteService.cpp:245: MiKTeX encountered an internal error.()
